runtime/commonMain/src/kotlinx/serialization/json/Json.kt

```diff
@@ -173,48 +107,48 @@ public constructor(
         override fun <T> decodeFromString(deserializer: DeserializationStrategy<T>, string: String): T =
             jsonInstance.decodeFromString(deserializer, string)
 
-        /**
-         * @see Json.encodeToJsonElement
-         */
-        public fun <T> encodeToJsonElement(serializer: SerializationStrategy<T>, value: T): JsonElement {
+        override fun <T> encodeToJsonElement(serializer: SerializationStrategy<T>, value: T): JsonElement {
             return jsonInstance.writeJson(value, serializer)
         }
+    }
+}
 
-        /**
-         * @see Json.encodeToJsonElement
-         */
-        public inline fun <reified T : Any> encodeToJsonElement(value: T): JsonElement {
-            return encodeToJsonElement(context.getContextualOrDefault(), value)
-        }
-
-        /**
-         * @see Json.parseToJsonElement
-         */
-        public fun parseToJsonElement(string: String): JsonElement {
-            return decodeFromString(JsonElementSerializer, string)
-        }
-
-        /**
-         * @see Json.decodeFromJsonElement
-         */
-        public fun <T> decodeFromJsonElement(deserializer: DeserializationStrategy<T>, json: JsonElement): T {
-            return jsonInstance.readJson(json, deserializer)
-        }
+/**
+ * Default Json constructor .
+ * To configure Json format behavior while still using only stable API it is possible to use `JsonConfiguration.copy` factory:
+ * ```
+ * val json = Json(configuration: = JsonConfiguration.Stable.copy(prettyPrint = true))
+ * ```
+ */
+public fun Json(configuration: JsonConfiguration = JsonConfiguration.Stable, context: SerialModule = EmptyModule
+): Json = JsonImpl(configuration, context)
 
-        /**
-         * @see Json.decodeFromJsonElement
-         */
-        public inline fun <reified T : Any> decodeFromJsonElement(tree: JsonElement): T =
-            decodeFromJsonElement(context.getContextualOrDefault(), tree)
-    }
+/**
+ * DSL-like constructor for [Json].
+ */
+public fun Json(block: JsonBuilder.() -> Unit): Json = JsonImpl(JsonBuilder().apply { block() })
```

Comment:
Constructor refactoring is postponed to next PR?

Reply:
Indeed